rosa_govcloud/rosa-govcloud-account-management.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-govcloud-account-management"]
+= Managing your ROSA AWS GovCloud account
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-govcloud-account-management
+
+toc::[]
+
+//I'm not sure what I meant by <Govcloud statement>, but there was likely a request for making a statement about the access or similar to it. I'll see what I can find
+//[NOTE]
+//====
+//<Govcloud statement>
+//====
+
+Once you have access to the VPN and FedRAMP accounts, you can manage the credentials as needed.
```

Comment:
This reference to VPN is still in the preview - will it be removed from the final rendering somehow?

Reply:
missed that one thanks, removed now, let me know when you would like it merged to go live as I am unsure of the live date for just this stage of getting started and installing the cluster, thanks